vocode/streaming/streaming_conversation.py

```diff
@@ -273,6 +273,10 @@ async def process(self, transcription: Transcription):
                 )
                 self.has_associated_unignored_utterance = not transcription.is_final
                 if self.conversation.current_transcription_is_interrupt:
+                    if self.conversation.transcript.get_last_user_message() is not None:
```

Comment:
trying to think what the most useful log is - last message is useful but I think the `Transcription` that invokes the worker process is even more useful, including the content + confidence.